client/packages/config/src/config.ts

```diff
@@ -1,31 +1,39 @@
 declare const API_HOST: string;
+declare const PLUGIN_URL: string;
 
 // For production, API is on the same domain/ip and port as web app, available through sub-route
-// i.e. web app is on https://my.openmsupply.com/, then graphql will be available https://my.openmsupply.com/graphql 
-// and files on https://my.openmsupply.com/files 
+// i.e. web app is on https://my.openmsupply.com/, then graphql will be available https://my.openmsupply.com/graphql
+// and files on https://my.openmsupply.com/files
 
-// For development, API server and front end are launched seperately on different ports and possible different IPs
+// For development, API server and front end are launched separately on different ports and possible different IPs
 // by default we assume development API server is launched on the same domain/ip and on port 8000. We can overwrite this
 // with API_HOST which is available through webpack.DefinePlugin (i.e. webpack server --env API_PORT=800 --env API_IP 'localhost')
 
-// Important to note, if we overwrite API_HOST in development, we should use ip/domain that is known outsideof localhost
-// because web app in development mode may be accessed by clients on different machine (i.e. when debugin Android app)
+// When developing plugins, you can set the env variable PLUGIN_URL to the url of the plugin server to enable hot reloading
+// e.g. `webpack-cli serve --port 3000 --env PLUGIN_URL=http://localhost:3000`,
```

Comment:
What is the plugin server? the remote server?

Reply:
ah, it should be the local webpack dev server of the plugin app. I've updated the comment - thanks